src/contexts/LayoutContext/state.ts

```diff
@@ -10,8 +10,14 @@ export interface LayoutState {
 }
 
 const INITIAL_LAYOUT_STATE: LayoutState = {
-  sideNav: {},
-  topNav: {},
+  sideNav: {
```

Comment:
a default for all values doesnt seem right, because there can be situations where states like `hasCompactMode`, `isToggled`, can be true on initial load. Having empty object would let the consumers know that it is not initialized yet.

Reply:
but we were using undefined as a falsy value. In that case it should explicitly be initialized with false

Reply:
Yes, the decision to treat `undefined` as `false` or to differentiate (like, stay in a loading state) should be in the consuming component.

Reply:
ok